about.md

```diff
@@ -52,38 +52,36 @@ University of Michigan Advanced Research Computing is an Associate member of the
 
 <h4>Current OSiRIS Staff</h4>
 
-| Ezra Kissel | | | | IU Lead Research Scientist |
 | Jeremy Musser | | | | IU Grad Student Engineer |
 | Alex Shroyer | | | | IU Engineer | 
-| Jeff Goeke-Smith | | | | MSU Network Engineer |
 | Andy Keen | | | | MSU Lead Engineer |
 | Nick Rahme | | | | MSU Engineer | 
-| Jerome Kinlaw | | | | UM Engineer | 
+| Adam Pitcher | | | | MSU Lead Engineer |
 | Dan Kirkland | | | | UM Network Engineer | 
-| Benjeman Meekhof | | | | UM Lead Engineer |
 | Matt Lessins | | | | WSU Network Engineer |
 | Carlo Musante | | | | WSU Network Engineer |
 | Michael Thompson | | | | WSU Lead Engineer |
 
-<h4>OSiRIS Undergraduate Students</h4>
```

Comment:
Don't delete this header
Add the Indiana students

Reply:
I added back the header and added the 2 IU students under it.